docs/user-guide/contextual-assistants.rst

```diff
@@ -856,7 +613,7 @@ We define some intent mappings to make it more intuitive to the user what an int
 Out of scope intent
 """""""""""""""""""
 
-It is good practice to also handle questions you know your users may ask, but you don’t necessarily have a skill
+It is good practice to also handle questions you know your users may ask, but for which you don’t necessarily have an action 
```

Comment:
i'm not sure I'd say action. Because it's not necessarily an action. Maybe we can say "user goal" ? https://rasa.com/docs/rasa/glossary/

Reply:
as in "for which you don't necessarily have a user goal"? Or "handle user goals you know your users might have"?

Reply:
yeah i would say "handle user goals"